src/core_plugins/kibana/server/routes/api/home/register_tutorials.js

```diff
@@ -22,8 +22,8 @@ export function registerTutorials(server) {
   server.route({
     path: '/api/kibana/home/tutorials',
     method: ['GET'],
-    handler: async function (req, reply) {
-      reply(server.getTutorials());
+    handler: async function () {
+      return server.getTutorials();
```

Comment:
I'm not sure if `server.getTutorials()` returns a promise here, but if it was using `return await` I would know for sure, and if it's not then there's no need for this to be an `async` function.